app/pages/cif/project-revision/new.tsx

```diff
@@ -0,0 +1,127 @@
+import { Button } from "@button-inc/bcgov-theme";
+import FormBase from "components/Form/FormBase";
+import SelectRfpWidget from "components/Form/SelectRfpWidget";
+import DefaultLayout from "components/Layout/DefaultLayout";
+import projectSchema from "data/jsonSchemaForm/projectSchema";
+import { JSONSchema7, JSONSchema7TypeName } from "json-schema";
+import withRelayOptions from "lib/relay/withRelayOptions";
+import EmptyObjectFieldTemplate from "lib/theme/EmptyObjectFieldTemplate";
+import { useCreateProjectMutation } from "mutations/Project/createProject";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { graphql, usePreloadedQuery } from "react-relay/hooks";
+import { RelayProps, withRelay } from "relay-nextjs";
+import { getProjectRevisionFormPageRoute } from "routes/pageRoutes";
+import { newProjectRevisionQuery } from "__generated__/newProjectRevisionQuery.graphql";
+
+const schema: JSONSchema7 = {
+  $schema: projectSchema.$schema,
+  type: projectSchema.type as JSONSchema7TypeName,
+  required: ["fundingStreamRfpId"],
+  properties: {
+    fundingStreamRfpId: {
+      type: "number",
+      title: "RFP Year ID",
+    },
+  },
+};
+
+const uiSchema = {
+  fundingStreamRfpId: {
+    "ui:widget": "SelectRfpWidget",
+    "ui:options": {
+      text: `text`,
+      label: true,
+    },
+  },
+};
+
+export const pageQuery = graphql`
+  query newProjectRevisionQuery {
+    query {
+      session {
+        ...DefaultLayout_session
+      }
+      ...SelectRfpWidget_query
+    }
+  }
+`;
+
+export function ProjectRevisionNew({
+  preloadedQuery,
+}: RelayProps<{}, newProjectRevisionQuery>) {
+  const { query } = usePreloadedQuery(pageQuery, preloadedQuery);
+
+  const router = useRouter();
+  const [formState, setFormState] = useState({});
```

Comment:
If you don't pass `formData` into `FormBase`, [the form clears whenever the state changes](https://github.com/rjsf-team/react-jsonschema-form/issues/584) (in this case, most relevantly when you click `Submit`). We can't grab `formData` from a graphql query like we do in the other forms because the project doesn't get a `projectId` until it's created (you click `Confirm`).